apps/labrinth/src/lib.rs

```diff
@@ -425,6 +425,8 @@ pub fn check_env_vars() -> bool {
     failed |= check_var::<String>("SMTP_USERNAME");
     failed |= check_var::<String>("SMTP_PASSWORD");
     failed |= check_var::<String>("SMTP_HOST");
+    failed |= check_var::<u16>("SMTP_PORT");
+    failed |= check_var::<bool>("SMTP_TLS");
```

Comment:
Now that `SMTP_TLS` accepts a string value, this check should be updated accordingly:

```suggestion
    failed |= check_var::<String>("SMTP_TLS");
```

While this doesn't impact functionality, it gets rid of an incorrect warning during Labrinth startup, which is always a nice improvement.